src/ckb/channel.rs

```diff
@@ -3827,6 +3884,9 @@ impl ChannelActorState {
                 )
             };
 
+            // FIXME(yukang): how do we make sure time_locked_ckb_amount >= commitment_tx_fee ?
+            time_locked_ckb_amount -= commitment_tx_fee;
```

Comment:
TODO: seems we need to add a check to make sure `reserved ckb amount >= commitment_tx_fee`, but when we're opening a channel, which side is the `to_local`? @quake